### DISCUSSION TITLE
Azure symbol (from sample az segment) not in my font?

### DISCUSSION THREAD

**Discussion by SvenAelterman** [JanDeDobbeleer/oh-my-posh#4095] [Troubleshoot] [answered]
Hi,
I am using the CaskaydiaCove nerd font and the Azure symbol that's shown in the sample az segment doesn't render.
Is it not a standard glyph in the nerd fonts?

**Comment by JanDeDobbeleer** [comment]
@SvenAelterman is the font and oh-my-posh up-to-date?

**Reply by SvenAelterman** [reply]
Oh-my-posh is at 18.0.1. I just downloaded CaskaydiaCove again from nerdfonts.com. Still no luck. (The glyph isn't even showing in my editor.)

**Reply by SvenAelterman** [reply, marked as answer]
I got the (current) Az logo using `\uebd8`.
